apps/condo/domains/acquiring/schema/ExportPaymentsService.js

```diff
@@ -77,6 +77,7 @@ const ExportPaymentsService = new GQLCustomSchema('ExportPaymentsService', {
 
                 const timeZone = normalizeTimeZone(timeZoneFromUser) || DEFAULT_ORGANIZATION_TIMEZONE
                 const formatDate = (date) => dayjs(date).tz(timeZone).format(DATE_FORMAT)
+                const isInvoicePayments = !!get(where, 'invoice.organization.id')
```

Comment:
best to specify fallback value as something that will be false always, here is an example:

`!!get(where, 'invoice.organization.id', null)`